### ISSUE TITLE
Something went wrong creating your account

### ISSUE THREAD

**Issue by Ashish117** [FeezyHendrix/Insta-mass-account-creator#124]
When running Creator.py it will open browser and input everything and when signing up it will say "Something went wrong creating your account". It will save details to usernames.pkl but when logging in with that information says "Your account has been disabled for violating our terms: http://instagram.com/about/legal/terms/"

Os - macOS
Browser: Chrome

**Comment by williamyorkl** [comment]
meeting the same issue, still looking for luck, as I've try tons of ways, chaning ip etc,but it still fail.....

**Comment by Ashish117** [comment]
FeezyHendrix told me that may be an issue with proxy.

**Comment by williamyorkl** [comment]
yeah, firstly,I try different IP,and it worked for few days using the same chromedriver, then the other day,all the ip went invaild,that's pretty weird, so I use the same ip tried on the cellphone,then it worked again,so I assume that the ip and the environment both cause the issue, so better use several drivers, may I know if you are still working on this ? or solved ?

**Comment by williamyorkl** [comment]
update : 
try again just now, no much luck for changing another driver, I use opera ,and still fail, but the IP works when I sign up on android ...maybe can try on the android ?

**Comment by JordySteyaert** [comment]
IG only allows you to create 5 accounts per device. I think this is the issue.

**Comment by williamyorkl** [comment]
that's weird,cuz I've created hundreds of accounts in the same computer for almost a week...and when it come the other day,it fails at all



------------------ 原始邮件 ------------------
发件人: "JordySteyaert"<notifications@github.com>; 
发送时间: 2019年9月17日(星期二) 晚上10:13
收件人: "FeezyHendrix/Insta-mass-account-creator"<Insta-mass-account-creator@noreply.github.com>; 
抄送: "梁永鹏"<p.1403303507@qq.com>; "Comment"<comment@noreply.github.com>; 
主题: Re: [FeezyHendrix/Insta-mass-account-creator] Something went wrongcreating your account (#124)




IG only allows you to create 5 accounts per device. I think this is the issue.

**Comment by Ashish117** [comment]
> that's weird,cuz I've created hundreds of accounts in the same computer for almost a week...and when it come the other day,it fails at all
> […](#)

Are you able to create account now or using proxy?

**Comment by williamyorkl** [comment]
Yes,still Available, but the proxies is pretty expensive, and after creating the account, have to verify a phone number, or else go to challenge page ASAP .....so sad still&nbsp;



发自我的iPhone


------------------ Original ------------------

**Comment by Ashish117** [comment]
So after creating account we need to manually verify phone number? If so
why we use this script!

**Comment by williamyorkl** [comment]
there could be a phone number pool ,and charge the fee for per message....tried, it will still fail, because once ig detect that your network is unstable, then it require you verify your phone again, you have to prove the same phone number, or else will still get ban




------------------ 原始邮件 ------------------
发件人: "Ashish117"<notifications@github.com>;
发送时间: 2019年9月21日(星期六) 凌晨0:58
收件人: "FeezyHendrix/Insta-mass-account-creator"<Insta-mass-account-creator@noreply.github.com>;
抄送: "梁永鹏"<p.1403303507@qq.com>;"Comment"<comment@noreply.github.com>;
主题: Re: [FeezyHendrix/Insta-mass-account-creator] Something went wrongcreating your account (#124)



So after creating account we need to manually verify phone number? If so
 why we use this script!

**Comment by Ashish117** [comment]
I have a question that when should we verify phone number? Within 24hrs. So
can we do anything with that account in 24hrs i mean do likes, follow etc?

**Comment by Klapkaak078** [comment]
Maybe you can add use a mobile emulator, And change from device number. 👌

**Comment by Ashish117** [comment]
> Maybe you can add a mobile emulator, And change from device number. 👌

Explain!

**Comment by williamyorkl** [comment]
haha,seems that will be a good solution, but how to combine the selenium with the mobile emulator, that will be a problem, maybe try use a emulator with appium? like mobile automation test, and also the IP are still of great importance as well....




------------------ 原始邮件 ------------------
发件人: "Ashish117"<notifications@github.com>; 
发送时间: 2019年9月27日(星期五) 中午11:22
收件人: "FeezyHendrix/Insta-mass-account-creator"<Insta-mass-account-creator@noreply.github.com>; 
抄送: "梁永鹏"<p.1403303507@qq.com>; "Comment"<comment@noreply.github.com>; 
主题: Re: [FeezyHendrix/Insta-mass-account-creator] Something went wrongcreating your account (#124)



 
Maybe you can add a mobile emulator, And change from device number. 
  
Explain!
